tensorboard/webapp/widgets/histogram/histogram_test.ts

```diff
@@ -1236,6 +1243,54 @@ describe('histogram test', () => {
 
         expect(onLinkedTimeToggledSpy).toHaveBeenCalledOnceWith();
       });
+
+      it('emits linked time change event when fob is dragged in single selection', () => {
+        const fixture = createComponent('foo', [
+          buildHistogramDatum({step: 0, wallTime: 100}),
+          buildHistogramDatum({step: 5, wallTime: 400}),
+          buildHistogramDatum({step: 10, wallTime: 400}),
+          buildHistogramDatum({step: 40, wallTime: 400}),
+        ]);
+        const onLinkedTimeSelectionChangedSpy = jasmine.createSpy();
+        fixture.componentInstance.timeSelection = {
+          start: {step: 0},
+          end: null,
+        };
+        fixture.componentInstance.onLinkedTimeSelectionChanged =
+          onLinkedTimeSelectionChangedSpy;
+        fixture.detectChanges();
+        intersectionObserver.simulateVisibilityChange(fixture, true);
+        const testController = fixture.debugElement.query(
+          By.directive(CardFobControllerComponent)
+        ).componentInstance;
+        const fobStartPosition = testController.root.nativeElement
+          .querySelector('.time-fob-wrapper')
+          .getBoundingClientRect().top;
+
+        // Simulate dragging fob to step 10.
+        testController.startDrag(Fob.START, TimeSelectionAffordance.FOB);
+        const fakeEvent = new MouseEvent('mousemove', {
+          clientY: 5 + fobStartPosition, // Add the difference between step 5 and 10, which is equal to 5.
+          movementY: 1,
+        });
+        testController.mouseMove(fakeEvent);
+        testController.stopDrag();
+        fixture.detectChanges();
+
+        expect(onLinkedTimeSelectionChangedSpy).toHaveBeenCalledWith({
```

Comment:
For sanity, would be worth saying first action is result of call to mouseMove() while second is action is result of call to stopDrag().